internal/docker/builder.go

```diff
@@ -468,13 +484,18 @@ func createNetwork(docker *client.Client, blueprintName string) (networkID strin
 		},
 	})
 	if err != nil {
-		log.Printf("Failed to create docker network: %s\n", err)
-		return ""
+		return "", fmt.Errorf("%s: failed to create docker network. %w", blueprintName, err)
 	}
 	if nw.Warning != "" {
+		if nw.ID == "" {
+			return "", fmt.Errorf("%s: fatal warning while creating docker network. %w", blueprintName, err)
```

Comment:
`err` will be `nil` here, you probably meant to log `nw.Warning` instead.

Reply:
Good catch!